src/FixMessageUtil.h

```diff
@@ -114,6 +114,7 @@ class FixMessageUtil {
 						addFixGroups(group, entry);
 
 						map->addGroup(group->field(), *group);
+            delete group;
```

Comment:
We're creating a new Group within a code block and then we set fields based on that.  I'm pretty sure quickfix is copying values (vs. setting a pointer), so we need to clean it up.

```					
					if(entry->Has(groupKey) || entry->Has(tagKey)) {

						FIX::Group* group = new FIX::Group(
							groupObj->Get(indexKey)->ToInteger()->Value(),
							groupObj->Get(delimKey)->ToInteger()->Value());
				
						addFixTags(group, entry);
						addFixGroups(group, entry);

						map->addGroup(group->field(), *group);
                        delete group;
				
					} else {

```
Plus I did see a leak for it reported by valgrind.  Just having a hard time finding it now in all of the logs.